src/modules/blocks/rsvp/container-header/template.js

```diff
@@ -102,53 +45,61 @@ const getCapacityLabel = ( capacity ) => {
 
 const RSVPContainerHeader = ( {
 	description,
-	hasAttendeeInfoFields,
+	isAddEditOpen,
 	isCreated,
-	isDisabled,
-	isSelected,
-	onTempDescriptionChange,
-	onTempTitleChange,
-	tempDescription,
-	tempTitle,
 	title,
 	available,
+	setAddEditOpen,
 } ) => {
+
+	if ( isAddEditOpen ) {
+		return null;
+	}
+
+	const leftSide = (
```

Comment:
Nit: Not sure if `side` would be too generic, I think we would normally use column, you can maybe double check with Stephen. It's just that I didn't see us using that term for layouts, I think we would normally use columns.

Reply:
I've renamed this to leftColumn and rightColumn